[draft] Case studies (#46416)
mui/material-ui: docs/pages/customers/delivery-hero.md

---
title: Delivery Hero
description: How Delivery Hero's Partner Portal streamlined development with MUI X.
image: '/static/branding/companies/deliveryhero_spotlight.svg'
date: 2022-06-08T00:00:00.000Z
tags: ['MUI X']
rank: '7'
manualCard: true
---

# How Delivery Hero's Partner Portal streamlined development with MUI X

<img
    id="blog-responsive-image"
    src="/static/branding/companies/headers/deliveryhero-header.png"
    alt="Delivery Hero logo"
    style="width: 100%; height: auto; object-fit: cover; object-position: top left; border: 0px; margin-left: 0; margin-bottom: 20px; display: block; text-align: left;"
  />

## Overview

> [Delivery Hero](https://www.deliveryhero.com/) is a leading local delivery platform, operating in around 70 countries. connecting customers with restaurants, shops, and essential services through fast, technology-driven delivery solutions. Their Partner Portal provides restaurant and vendors with tools to manage operations efficiently, enhance customer experience, and optimize performance.

Reviewer: ```suggestion
[Delivery Hero](https://www.deliveryhero.com/) is a leading local delivery platform, operating in around 70 countries. They connect customers with restaurants, shops, and essential services through fast, technology-driven delivery solutions. Their Partner Portal provides restaurant and vendors with tools to manage operations efficiently, enhance customer experience, and optimizing performance.
```

<blockquote style="margin: 32px 0; padding: 24px 32px; background: var(--muidocs-palette-background-default); border-left: 6px solid var(--muidocs-palette-primary-main); color: #f5f5f7; border-radius: 8px; font-size: 1.15rem; font-style: italic;">
  By leveraging the MUI X Date Time Picker and Data Grid Pro, we have been able to provide our customers with cutting-edge, well-designed features in a shorter development time and with less effort.
  <br>
  <span style="display: block; margin-top: 12px; font-size: 1rem; font-style: normal; color: #bbb; font-weight: 500;">
    – Ahmed Ibrahim, Senior Manager Software Engineering, Vendor Growth
  </span>
</blockquote>
</span>

## Challenge

Before adopting MUI X, the Delivery Hero team faced two major challenges:

- Fixing vendor reported bugs and feature requests took time due to the slow development speed. Building, testing and refining UI components took too much time.
- Ensuring a consistent, easy-to-use and performant experience for end users was difficult.

## Solution

To address these challenges, the team integrated the **MUI X Pro Date and Time Pickers** and **Data Grid** into their platform.
These components enabled them to:

> - Accelerate development by using pre-built, high-quality components. It reduced the time spent on UI implementation.

Reviewer: ```suggestion
- Accelerate development by using pre-built, high-quality components. This reduced the time spent on UI implementation.
```

> - Enhance usability by offering a modern, sleek UI that improved user interactions and consistency.

Reviewer: ```suggestion
- Enhance usability by offering a modern, sleek UI that improves user interactions and consistency.
```

- Optimize performance by providing a smooth experience for restaurant and shop managers.

## Results

After implementing MUI X, the team saw immediate benefits. It gave them more time to focus on other business features. UI development became faster and more efficient.

One key success story was the implementation of the Date Time Picker, which significantly reduced the number of clicks required for users to achieve their desired results.
This simple enhancement greatly improved user efficiency and satisfaction.

## Developer experience

> The team praised MUI documentation, describing it as clear, well-structured, and easy to follow, which helped their developers integrate and customize components effortlessly.

Reviewer: ```suggestion
The team praised MUI documentation, describing it as clear, well-structured, and easy to follow. This helped their developers integrate and customize components effortlessly.
```
